docs/src/tutorials/Add-New-Network.md

```diff
@@ -0,0 +1,163 @@
+# Add New Network
+
+Parity Signer supports adding a Substrate-based network or update the existing network via QR code.
+
+After you've installed [required software](#Prerequisites), you need to add *Network's Spec* to Signer and add *Network Metadata* for this network, so that Signer is able to decode, and you could read and verify transactions you are signing on this network.
+
+If you are updating metadata for already existing network you only need to update *Network Metadata*.
+
+s*Network Specs*
+
+1. [Get](#get-network-spec)
+2. [Sign](#sign-network-spec)
+3. [Feed into Signer](#feed-network-spec-into-signer)
+
+*Network Metadata*
+
+4. [Get](#get-network-metadata)
+5. [Sign](#sign-network-metadata)
+6. [Feed into Signer](#feed-network-metadata-into-signer)
+
+### Prerequisites
+
+- [ ] Websocket URL of the network (we maintain an [address book](https://github.com/paritytech/parity-signer/blob/master/rust/generate_message/address_book))
```

Comment:
`address_book` is not exactly maintained at the moment. I wrote it for tests a few month ago and many things have changed since. It has some possibly confusing comments and most importantly no network names and only one url for each network. Source was polkadot.js, we should refer users there.

Reply:
Oh I see, I could link this https://github.com/polkadot-js/apps/tree/master/packages/apps-config/src/endpoints or remove mentions of the examples completely. Wdyt?